Drivers/Inc/CANbus.h

```diff
@@ -46,9 +46,16 @@ typedef struct {
 /**
  * @brief   Initializes the CAN system
  * @param   loopback	: if we should use loopback mode (for testing)	
+ * @param   faultState  : fault state determines whether to implement Rx and Tx interrupts 
```

Comment:
should write what values mean what

Reply:
fixed

Reply:
I mean `false means interrupts are not enabled` or something like that

Reply:
fixed